learning-objectives/intl/pt.yml

```diff
@@ -1949,6 +1949,151 @@ ai-prompting/open-ai-api:
+    - title: Conceito de valores fora do alcance
+      url: https://docs.google.com/document/d/1-SsxH5Ke3qAaWg5YKrkRwkExd91Q0RdJGplLAiXPiYA/edit?usp=sharing
+    - title: Versão em áudio
+      url: https://drive.google.com/file/d/1Z963-z4MCiRRjt5RZF75oH_CH1BqDNrO/view?usp=drive_link
+    - title: Como utilizar EXCEPT para selecionar variáveis
+      url: https://www.loom.com/share/2fa18d5cfd33403faaa794a2b08671f1?sid=2d16f30b-a26c-4d3c-bfd8-88ba7c664f25
+    - title: Como utilizar comandos LIKE e REGEXP
+      url: https://www.loom.com/share/3f7bb1362d5947ecb8c83e6f0690a4a5?sid=76f52b28-efb3-4991-b4c1-bd288bd4e805
+    - title: Documentação Google sobre Funções de String
+      url: https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions
+    - title: Conceito de variáveis categóricas
+      url: https://docs.google.com/document/d/1BIqa7xR0JNadM9kQExypo3plnwrNia92zCuxkPuMj5A/edit?usp=sharing
+    - title: Conceito de variáveis categóricas (versão em áudio)
+      url: https://drive.google.com/file/d/13gQ6Ni9FUfDmi5bjQivjeKj04fAI8Gl2/view?usp=drive_link
+    - title: Vídeo sobre tipos de variáveis
+      url: https://www.loom.com/share/afa56686810e46309761bc889364b2d9?sid=cabf587c-7ba1-4419-b501-32c98a73e191
+    - title: Identificar valores discrepantes em variáveis numéricas com MAX, MIN e AVG
+      url: https://www.loom.com/share/4fb40d01dddf4e3cbf981029ed08613a?sid=5e46b20b-5ea4-47c4-a808-99969a369431
+    - title: Conceito de variáveis numéricas
+      url: https://docs.google.com/document/d/1oTn46djgWYlhcY9_Ak9H2edY2_rR8gpJco66njPwkCc/edit?usp=sharing
+    - title: Tipos de dados em SQL
+      url: https://docs.google.com/document/d/1z0evGu8JtjDPuFlH1pkbqYR6scRwukPfwWKJ-KmZRmI/edit?usp=sharing
+    - title: Tipos de dados em SQL
+      url: https://www.youtube.com/watch?v=A1tZ-d2uxws
+    - title: Alterar tipo de dado com CAST
+      url: https://www.loom.com/share/dfca15dd59084f83a1343eb476c71e96?sid=5ffd1896-5cf2-4ad5-88dc-741152483c45
+    - title: Conceito de criar novas variáveis
+      url: https://docs.google.com/document/d/1yp5MVB8iirIbYwptscJ4YrjsY4Bnd8mrsDB75Onun2s/edit?usp=sharing
+    - title: Versão em áudio
+      url: https://drive.google.com/file/d/15P7Gp4_F_EOasIgW7SUhK8bf0MIDD6K8/view?usp=drive_link
+    - title: Como criar novas variáveis com CONCAT e operações matemáticas
+      url: https://www.loom.com/share/65e15aa134c545bc9cfdc6b88bc2ae78?sid=46988f46-bcc0-471f-b89e-55c7ffe953b5
+    - title: Tipos de JOIN em SQL
+      url: https://www.youtube.com/watch?v=HDBdAIcqJQo
+    - title: Diferença entre VIEW e CREATE TABLE
+      url: https://www.loom.com/share/97dbe237b9de459baa4d788f8bb44502?sid=e2168d4f-a8ff-44e4-b3f0-55e72596350a
+    - title: Como utilizar comando WITH e para que serve
+      url: https://www.loom.com/share/8e2f6f70db214e2484af45375539d8e9?sid=222c7e6d-1ff5-430a-ab28-64c025f34ebf
+
```

Comment:
Sustituir por:

- title: Mini curso BigQuery
  url: https://coda.io/d/PT-Curso-Google-BigQuery_dvpJ2RzVDub/Configurando-o-Sandbox_suq00#_lu8z6
- title: Como começar com BigQuery (sandbox - versão gratuita)
  url: https://cloud.google.com/bigquery/docs/sandbox?hl=pt
- title: O que é BigQuery
  url: https://www.youtube.com/watch?v=BRPUA0EgS4I&list=PL5TJqBvpXQv5n1N15kcK1m9oKJm_cv-m6
- title: Como criar um Projeto, Conjunto de Dados e carregar tabela no BigQuery
  url: https://www.youtube.com/watch?v=z32438Yehl4&list=PL5TJqBvpXQv5n1N15kcK1m9oKJm_cv-m6&index=2
- title: O que é uma consulta (query)
  url: https://docs.google.com/document/d/1QjuWNUrCCs0kSMjq0RxnUoa8lCr58GvIwerYrLZs2Bs/edit?usp=sharing
- title: Como salvar queries
  url: https://www.loom.com/share/4140a54ac8c845d781058d8f0bb53e30?sid=e8e3bb4f-d70b-44f8-a79b-377ded6a88a1
- title: Como identificar nulos usando COUNT, WHERE e IS NULL
  url: https://www.loom.com/share/972220ee18874d1b872c206f5e62627c?sid=fb520257-c035-490a-8d42-c41d134476c6
- title: Como identificar duplicados através de GROUP BY e HAVING (Veja também a diferença entre os comandos WHERE e HAVING)
  url: https://www.loom.com/share/7f066ca3cad3438180509ba0643221a5?sid=1555e2cc-a68c-4ff8-80e4-2133059f0acb
- title: Como usar EXCEPT para selecionar variáveis
  url: https://www.loom.com/share/2c507731f3004d9f8f64c555232702a3?sid=f608d12d-d248-47e5-b034-15259817cf5d
- title: Como usar os comandos LIKE e REGEXP
  url: https://www.loom.com/share/3ec6dda6ef4741288bda6044b50b0cbc?sid=a47774e0-e3c5-4dd4-b5f3-d3096d5bd70d
- title: Documentação do Google sobre funções de string
  url: https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions
- title: Conceito de variáveis ​​categóricas
  url: https://docs.google.com/document/d/1cOTKw04u2008G7JyRaYHzSzyWRjJwxLryn8hIxkO2es/edit?usp=sharing
- title: Vídeo sobre tipos de variáveis
  url: https://www.youtube.com/watch?v=c2R378U4gcA
- title: Identifique valores discrepantes em variáveis ​​numéricas com MAX, MIN e AVG
  url: https://www.loom.com/share/86e9b5b4cbe941348193039b0c60691f?sid=1472fe57-e4fd-4287-8408-405c8c922b8e
- title: Conceito de variáveis ​​numéricas
  url: https://docs.google.com/document/d/1-C065f5Vti_H7vO1ioqHN0IYocxz5hVpCruc0RFnRa8/edit?usp=sharing
- title: Tipos de dados em SQL
  url: https://docs.google.com/document/d/1GYwd_Gg_gsIJuv0XnJ7C7JMorYNom9Z7wyfLPclHTBo/edit?usp=sharing
- title: Tipos de dados em SQL (vídeo)
  url: https://www.youtube.com/watch?v=X40zYYRsw1U
- title: Alterar tipo de dados com CAST
  url: https://www.loom.com/share/93e9005ecfb944ddbe9effa8dcb2fc27?sid=ab42f8e3-6d41-453b-b080-7461e3736582
- title: Como criar novas variáveis ​​com CONCAT e operações matemáticas
  url: https://www.loom.com/share/9f8a444c394e40dfb2cd693b2ac4201f?sid=004aeaae-97e1-41b2-8cbd-26d45ff48cc8
- title: Tipos de JOIN em SQL
  url: https://www.youtube.com/watch?v=3IdKyi-KezU
- title: Diferença entre VIEW e CREATE TABLE
  url: https://www.loom.com/share/bd6987ebe2ad48eebf291931981a0c4d?sid=ca3e7ea1-bc44-4f0a-b332-36f86b093f5f
- title: Como utilizar comando WITH e para que serve
  url: https://www.loom.com/share/2d982ea5dd58420e9e3fb15b9153f6c8?sid=6fc75c6e-f68a-43f9-9f94-8ff494432f69